src/encoded/static/components/dbxref.js

```diff
@@ -369,8 +380,10 @@ DbxrefList.propTypes = {
     dbxrefs: PropTypes.array.isRequired, // Array of dbxref values to display
     context: PropTypes.object.isRequired, // Object containing the dbxref
     addClasses: PropTypes.string, // CSS class to apply to dbxref list
+    title: PropTypes.string,
```

Comment:
Could you add a comment here too, and give a reason for its existence, like “Displayed instead of Dbxref string” or something like that. Same for its occurrence above.